pkg/deps/config/suppliers.go

```diff
@@ -617,30 +617,32 @@ func SupplyMiner(
 	return depinject.Configs(deps, depinject.Supply(mnr)), nil
 }
 
-// SupplyRelayMeter constructs a RelayMeter instance and returns a new depinject.Config with it supplied.
+// SupplyRelayMeterFn returns a function which constructs a RelayMeter instance
+// and returns a new depinject.Config with it supplied.
 //
-// - Supplies RelayMeter to the dependency injection config
-// - Returns updated config and error if any
+// - Accepts enableOverServicing flag for proxy setup
```

Comment:
I'm expecting for this to be part of `cmd_start.go` but can't find it in this function. Why is that?

Reply:
It is.

All the `SupplyXXX` are called from `deps.go#setupRelayerDependencies`